index.html

```diff
@@ -1,45 +1,223 @@
+                <p class="date3"><p>
+                    <div class="modal fade" id="myModal4" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+                      <div class="modal-dialog">
+                        <div class="modal-content">
+                            <div class="post4">
+                                <div class="modal-header">
+                                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                                    <div class="title3" class="modal-title" id="myModalLabel"></div>
+                                </div>
+                                <div class="modal-body">
+                                    <div class="content3 search15" ></div>
+                                    <center><img src="images/agujero.jpg" class="imagezise" alt="image of agujero" /></center>
+                                </div>
+                                <div class="modal-footer">
+                                    <br/>
+                                    <br/>
+                                    <br/>
+                                    <p class="date3"><p>
+                                    <p>Commentation</p>
+                                    <div class="coments4 coments"></div>
+                                    <div class="coment">
+                                        <input type="text" name="checkListItem4" maxlength="140" id="clcoment4" class="form-control" onfocus="javascript:this.value=''"/>
+                                        <div id="add4" class="botadd">Add!</div>
+                                        <br/>
+                                    </div>
+                                </div><!--End the modal footer-->
+                            </div><!--end the post 4 -->
+                        </div><!--End the modal-->
+                      </div><!--End the modal-->
+                    </div><!--End the modal-->
+            </div><!--end the fourth post -->
+                <hr/>
+                <p class="bottons search16" id="ocultar">Back  ←</p>
+                <br/>
+                <br/>
+                <hr/>
+        </div>
+        </div>
+        <p class="copyright">Copyright © Your Website 2015</p>
+    </div>
+        </div>
+        <!-- Bootstrap core JavaScript
+        ================================================== -->
+        <!-- Placed at the end of the document so the pages load faster -->
+        <script src="js/libs/jquery-2.1.1.js"></script>
+        <script src="js/script.js"></script>
+        <script src="js/libs/bootstrap.js"></script>
+        <script type="text/javascript" src="/js/jquery-1.4.2.js"></script>
+        <script type="text/javascript" src="/js/jquery-highlight.js"></script>
+    </body>
+</html>

```

Comment:
Excellent to have followed the good practices of programming, documenting your code you will serve in the future to remind that it has been done, you could document a little more to put in practice.